src/components/table/table.tsx

```diff
@@ -67,6 +93,18 @@ export default function Table(props: TableData) {
           </TableBody>
         </MuiTable>
       </TableContainer>
-    </Box>
+      {props.data.length > paginationThreshold && (
```

Comment:
Maybe you could assign `props.data.length > paginationThreshold` to a variable because you also used it when setting default value for `rowsPerPage`